juniper/src/macros/tests/union.rs

```diff
@@ -36,6 +44,16 @@ enum WithGenerics<T> {
 enum DescriptionFirst {
     Concrete(Concrete),
 }
+enum ResolversFirst {
+    Concrete(Concrete),
+}
+
+enum CommasWithTrailing {
+    Concrete(Concrete),
+}
+enum ResolversWithTrailingComma {
+    Concrete(Concrete),
+}
```

Comment:
Why were these added back?

Reply:
Oops, my mistake

Reply:
I don't see this fixed? Is this a bad merge?